docs/ref/models/querysets.txt

```diff
@@ -516,8 +516,8 @@ values
 
 .. method:: values(*fields)
 
-Returns a ``ValuesQuerySet`` — a ``QuerySet`` subclass that returns
-dictionaries when used as an iterable, rather than model-instance objects.
+Returns ``QuerySet`` that returns dictionaries when used as an iterable,
```

Comment:
Returns a QuerySet that returns dictionaries, rather than model instances, when used as an iterable.